README.rst

```diff
@@ -44,6 +44,25 @@ if you prefer.
 **NOTE:** Some playbooks require specific groups to be defined in your inventory. Please refer to
 ``hosts`` in the playbook you want to use to ensure you've got the proper groups defined.
 
+Where should I put variables?
+-----------------------------
+
+All variables should be defined in ``defaults/main.yml`` for the role they're primarily used in.
+If the variable you're adding can be used in multiple roles define it in ``defaults/main.yml``
+for both roles. If the variable can contain a reasonable default value that should work for all
+possible labs then define that value in ``defaults/main.yml`` as well.  If not, you should still
+default the variable to something, but make the tasks that use the variable either fail
+gracefully without that var or prompt the user to define it if it's mandatory.
```

Comment:
Depends on what the task is doing.  In some cases that task might be mandatory for the rest of the playbook to complete.